sysutils/kubectx/Portfile

```diff
@@ -0,0 +1,39 @@
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        ahmetb kubectx 0.6.3 v
+categories          sysutils
+platforms           darwin
+supported_archs     noarch
+maintainers         {breun.nl:nils @breun} openmaintainer
+license             Apache-2
+
+description         Power tools for kubectl
+long_description    kubectx helps you switch between clusters back and forth. \
+                    kubens helps you switch between Kubernetes namespaces smoothly.
+
+checksums           rmd160  35712f70bc6729e5126f6ae83a76744edc6e8f59 \
+                    sha256  3bfab7c5c5d2285afbdb74a6f7f46ab55561135a87a4bb8fc7d79e17bfdbd080 \
+                    size    483475
+
+depends_run         port:bash-completion port:kubectl
+
+use_configure       no
+build {}
+
+destroot    {
+    xinstall -m 755 -d ${destroot}${prefix}/share/${name}
+    copy ${worksrcpath}/CONTRIBUTING.md \
+        ${worksrcpath}/LICENSE \
+        ${worksrcpath}/README.md \
+        ${destroot}${prefix}/share/${name}
+
+    copy ${worksrcpath}/kubectx \
+        ${worksrcpath}/kubens \
+        ${destroot}${prefix}/bin
+
+    xinstall -d ${destroot}${prefix}/etc/bash_completion.d
+    copy ${worksrcpath}/completion/kubectx.bash \
+        ${worksrcpath}/completion/kubens.bash \
+        ${destroot}${prefix}/etc/bash_completion.d
```

Comment:
Instead of these copy commands, use xinstall per the [guide](https://guide.macports.org/#reference.tcl-extensions). This is the syntax you want.

`xinstall [-o owner] [-g group] [-m mode] [-W dir] [file1 file2 ...] directory`

Reply:
That guide actually first mentions the `copy` macro as the shorthand for `file copy` and says the macros should be preferred to using Tcl `file` commands directly, because they may work around certain bugs.

I don't see an explanation of why using `xinstall` should be preferred over `copy`. Why should `xinstall` be used instead of `copy`?

Reply:
That's a good question and I don't have a good answer about that. My thinking was that multiple files can be handled with xinstall in a single command. But you are right, copy is a completely valid.